Gemfile

```diff
@@ -6,7 +6,7 @@ git_source(:github) { |repo_name| "https://github.com/#{repo_name}.git" }
 
 gem 'dotenv-rails', require: 'dotenv/rails-now' # Load ENV vars before other gems
 
-gem 'rails', '~> 6.1.4'
+gem 'rails', '>= 6.1.4'
```

Comment:
Since we usually don't downgrade, should we just drop this restriction then?

Reply:
We can bump it to `~> 7` soon. Or do you mean remove the restriction entirely?

Reply:
Yeah, remove it entirely. It's almost the same as `>=`. The only downside I can think of is that bundler may go through a bigger pool of versions for resolutions but that's bundler's problem and only happens if we ran a blank `bundle update`.